README.md

```diff
@@ -12,19 +12,22 @@ On the host machine, install [Docker](https://docs.docker.com/get-docker/).
 
 ### DNS
 
-Update the corresponding hosts file for the local DNS
+Update the corresponding hosts file for the local DNS and add the following:
 
-#### Linux
 ```bash
-sudo vim /etc/hosts
-```
-and add the following
-
-```
 # Added for Keyman sites
 127.0.0.1       api.keyman.com.local 
 127.0.0.1       help.keyman.com.local 
 127.0.0.1       keyman.com.local
 127.0.0.1       keymanweb.com.local
 # End of section
```

Comment:
This shouldn't be needed if we can sort out why keyman-local.com is not resolving on your machine?

Reply:
> This shouldn't be needed if we can sort out why keyman-local.com is not resolving on your machine?

> But when I commented these out of my hosts file, they could not resolve.

Yes ... well I was suggesting using keyman-local.com. But if that's not going to be consistent, then we should include the .com.local or perhaps move to .com.localhost alternative, which should resolve on most modern machines without any dns needed.

Reply:
Yep, we'll go with *.com.localhost and avoid having to modify hosts files.